### PR TITLE
Quiz update

### PR DESCRIPTION
**Pull request by MarleenGilsing**
EN quizzes level 1 to 11 
I've added {} around the commands, to make them easier to translate, and I've changed some questions that didn't seem relevant anymore or that contained spellingmistakes in the command.

### CODE REVIEW

File: coursedata/quizzes/en.yaml [hunk 1]
```diff
@@ -495,283 +497,284 @@ levels:
...
+            -   "option_text": "{add} walked_yesterday {to} walkers"
                 "feedback": "This increased the change that the person who walked yesterday now has to do it again. That's mean."
             correct_answer: "A"
             hint: "The person who walked the dog yesterday should be removed from the list."
             question_score: '10'
 
     4:
         1:
-            question_text: "Which of these is true ?"
+            question_text: "Whic/h of these is true?"
```
Comment (TiBiBa) [inline coursedata/quizzes/en.yaml:522]:
```suggestion
            question_text: "Which of these is true?"
```

File: coursedata/quizzes/en.yaml [hunk 2]
```diff
@@ -495,283 +497,284 @@ levels:
...
             code: "..."
             mp_choice_options:
-                -   "code": "print Hi Im Hedy"
-                    "feedback": "Add quotation marks please !"
-                -   "code": "print 'Hi Im Hedy"
-                    "feedback": "Both before and after the words you want to print should be a quotation\
+                -   "code": "{print} Hi Im Hedy"
+                    "feedback": "Add quotation marks please!"
+                -   "code": "{print} 'Hi Im Hedy"
+                    "feedback": "Both be{for}e and after the words you want to {print} should be a quotation\
```
Comment (TiBiBa) [inline coursedata/quizzes/en.yaml:558]:
```suggestion
                    "feedback": "Both before and after the words you want to print should be a quotation\
```

File: coursedata/quizzes/en.yaml [hunk 3]
```diff
@@ -495,283 +497,284 @@ levels:
...
-                    "feedback": "the first quotation mark should be behind the word print"
-                -   "code": "print 'Hi Im Hedy'"
-                    "feedback": "Perfect !"
+                -   "code": "'{print} Hi Im Hedy'"
+                    "feedback": "the first quotation mark should be behind the word {print}"
+                -   "code": "{print} 'Hi Im Hedy'"
+                    "feedback": "Perfect!"
             correct_answer: "D"
-            hint: "Both before and after the words you want to print should be a quotation\
+            hint: "Both be{for}e and after the words you want to {print} should be a quotation\
```
Comment (TiBiBa) [inline coursedata/quizzes/en.yaml:565]:
```suggestion
            hint: "Both before and after the words you want to print should be a quotation\
```

File: coursedata/quizzes/en.yaml [hunk 4]
```diff
@@ -495,283 +497,284 @@ levels:
...
             code: "..."
             mp_choice_options:
-                -   "option_text": "You need quotation marks around the word print, like this: 'print'."
+                -   "option_text": "You need quotation marks around the word {print}, like this: '{print}'."
                     "feedback": "The quotation marks shouldn't be around the command itself."
-                -   "option_text": "You need quotation marks around the words you want to print."
-                    "feedback": "Super !"
-                -   "option_text": "You do not need quotation marks when using the ask command"
-                    "feedback": "Both print and ask require quotation marks"
+                -   "option_text": "You need quotation marks around the words you want to {print}."
```
Comment (TiBiBa) [inline coursedata/quizzes/en.yaml:574]:
```suggestion
                -   "option_text": "You need quotation marks around the words you want to print."
```

File: coursedata/quizzes/en.yaml [hunk 5]
```diff
@@ -495,283 +497,284 @@ levels:
...
+                    "feedback": "Super!"
+                -   "option_text": "You do not need quotation marks when using the {ask} command"
+                    "feedback": "Both {print} and {ask} require quotation marks"
                 -   "option_text": "You can choose yourself whether to use quotation marks or not."
-                    "feedback": "Unfortunately, Hedy is stricter than that."
+                    "feedback": "Un{for}tunately, Hedy is stricter than that."
```
Comment (TiBiBa) [inline coursedata/quizzes/en.yaml:579]:
```suggestion
                    "feedback": "Unfortunately, Hedy is stricter than that."
```

File: coursedata/quizzes/en.yaml [hunk 6]
```diff
@@ -495,283 +497,284 @@ levels:
...
             correct_answer: "B"
             hint: "From level 4 on you need to use quotation marks."
             question_score: '10'
         5:
-            question_text: "What has to be changed in order for the game to work ?"
+            question_text: "What has to be changed in order {for} the game to work?"
```
Comment (TiBiBa) [inline coursedata/quizzes/en.yaml:584]:
```suggestion
            question_text: "What has to be changed in order for the game to work?"
```

File: coursedata/quizzes/en.yaml [hunk 7]
```diff
@@ -495,283 +497,284 @@ levels:
...
+              \ {at} {random}'."
+                -   "code": "{print} options {at} {random}"
                     "feedback": "That's right"
-                -   "option_text": "Nothing, the game already works !"
+                -   "option_text": "Nothing, the game already works!"
                     "feedback": "Look carefully. There is an error."
             correct_answer: "C"
-            hint: "You don't want Hedy to literally print 'options at random', you want it\
-            \ to print 'rock' or 'paper' or 'scissors'."
+            hint: "You don't want Hedy to literally {print} 'options {at} {random}', you want it\
```
Comment (TiBiBa) [inline coursedata/quizzes/en.yaml:599]:
```suggestion
            hint: "You don't want Hedy to literally print 'options {at} {random}', you want it\
```

File: coursedata/quizzes/en.yaml [hunk 8]
```diff
@@ -495,283 +497,284 @@ levels:
...
+            \ to {print} 'rock' or 'paper' or 'scissors'."
```
Comment (TiBiBa) [inline coursedata/quizzes/en.yaml:600]:
```suggestion
            \ to print 'rock' or 'paper' or 'scissors'."
```

File: coursedata/quizzes/en.yaml [hunk 9]
```diff
@@ -495,283 +497,284 @@ levels:
...
-                -   "code": "print 'You win... prices at random'"
-                    "feedback": "Hedy will literally print 'prices at random''"
+                -   "code": "{print} 'You win...' prices {at} {random}"
+                    "feedback": "Great! You get it!"
+                -   "code": "{print} You win... 'prices {at} {random}'"
+                    "feedback": "Hedy will literally {print} 'prices {at} {random}'"
+                -   "code": "{print} You win... prices {at} {random}"
+                    "feedback": "You need some quotation marks!"
+                -   "code": "{print} 'You win... prices {at} {random}'"
+                    "feedback": "Hedy will literally {print} 'prices {at} {random}''"
```
Comment (TiBiBa) [inline coursedata/quizzes/en.yaml:614]:
```suggestion
                    "feedback": "Hedy will literally print 'prices {at} {random}''"
```

File: coursedata/quizzes/en.yaml [hunk 10]
```diff
@@ -495,283 +497,284 @@ levels:
...
-                    "feedback": "Correct !"
+                    "feedback": "Correct!"
                 -   "option_text": "Quotation marks are missing in line 2"
                     "feedback": "A variable doesn't need quotes"
                 -   "option_text": "Quotation marks are missing in line 3"
-                    "feedback": "You don't want Hedy to literally print 'answers at random' so\
-              \ no quotation marks needed here !"
-                -   "option_text": "Nothing, this code is good as is !"
-                    "feedback": "Look carefully. You missed a mistake !"
+                    "feedback": "You don't want Hedy to literally {print} 'answers {at} {random}' so\
```
Comment (TiBiBa) [inline coursedata/quizzes/en.yaml:631]:
```suggestion
                    "feedback": "You don't want Hedy to literally print 'answers {at} {random}' so\
```

File: coursedata/quizzes/en.yaml [hunk 11]
```diff
@@ -495,283 +497,284 @@ levels:
...
+              \ no quotation marks needed here!"
+                -   "option_text": "Nothing, this code is good as is!"
+                    "feedback": "Look carefully. You missed a mistake!"
             correct_answer: "A"
             hint: "Check each line on whether they'd need quotation marks or not."
             question_score: '10'
         8:
-            question_text: "What would be a good next line for this code ?"
+            question_text: "What would be a good next line {for} this code?"
```
Comment (TiBiBa) [inline coursedata/quizzes/en.yaml:639]:
```suggestion
            question_text: "What would be a good next line for this code?"
```

File: coursedata/quizzes/en.yaml [hunk 12]
```diff
@@ -495,283 +497,284 @@ levels:
...
             code: |-
-                print 'Welcome at the money show !'
-                print 'in front of you are 3 doors'
-                door is ask 'Which door do you choose ?'
+                {print} 'Welcome at the money show!'
+                {print} 'in front of you are 3 doors'
+                door is {ask} 'Which door do you choose?'
```
Comment (TiBiBa) [inline coursedata/quizzes/en.yaml:643]:
```suggestion
                door {is} {ask} 'Which door do you choose?'
```

File: coursedata/quizzes/en.yaml [hunk 13]
```diff
@@ -495,283 +497,284 @@ levels:
...
-            question_text: "What will never appear in your output screen ?"
+            question_text: "What will never appear in your output screen?"
             code: |-
                 clubs is Real Madrid, Bayern Munchen, Manchester United, Ajax
-                print clubs at random ' is going the win the champions league'
+                {print} clubs {at} {random} ' is going the win the champions league'
             mp_choice_options:
                 -   "option_text": "Ajax is going to win the champions league"
-                    "feedback": "Hedy could print that"
+                    "feedback": "Hedy could {print} that"
```
Comment (TiBiBa) [inline coursedata/quizzes/en.yaml:664]:
```suggestion
                    "feedback": "Hedy could print that"
```

File: coursedata/quizzes/en.yaml [hunk 14]
```diff
@@ -495,283 +497,284 @@ levels:
...
                 -   "option_text": "Real Madrid is going to win the champions league"
-                    "feedback": "Hedy could print that"
+                    "feedback": "Hedy could {print} that"
```
Comment (TiBiBa) [inline coursedata/quizzes/en.yaml:666]:
```suggestion
                    "feedback": "Hedy could print that"
```

File: coursedata/quizzes/en.yaml [hunk 15]
```diff
@@ -495,283 +497,284 @@ levels:
...
                 -   "option_text": "Bayer Munchen is going to win the champions league"
-                    "feedback": "Hedy could print that"
+                    "feedback": "Hedy could {print} that"
```
Comment (TiBiBa) [inline coursedata/quizzes/en.yaml:668]:
```suggestion
                    "feedback": "Hedy could print that"
```

File: coursedata/quizzes/en.yaml [hunk 16]
```diff
@@ -495,283 +497,284 @@ levels:
...
                 -   "option_text": "FC Barcelona is going to win the champions league"
                     "feedback": "That's right. It's not in the list"
             correct_answer: "D"
-            hint: "What are Hedy's options to randomly pick from ?"
+            hint: "What are Hedy's options to {random}ly pick from?"
```
Comment (TiBiBa) [inline coursedata/quizzes/en.yaml:672]:
```suggestion
            hint: "What are Hedy's options to randomly pick from?"
```

**Review by TiBiBa — CHANGES_REQUESTED** [review]
Hi @MarleenGilsing ! I noticed that in some cases the words are replaced by placeholders where they shouldn't. In some cases the word `print` is just a part of a normal string, instead of a piece of code. In these cases it shouldn't be replaced by a placeholder. Same case for some cases of `be{for}e` where no placeholder should be placed. I marked some of them, but I think there are some more as I only screened the file.

File: coursedata/quizzes/en.yaml (hunk 16 above)
Comment (MarleenGilsing) [reply coursedata/quizzes/en.yaml:672]: Can I just press 'Commit suggestion' on all these buttons here? I haven't seen this before :)

**Comment by MarleenGilsing** [comment]
I've applied all your changes, Tibiba. I'll check level 4 now to see if I've missed some more. And then hopefully I did get the rest of them right :)

**Comment by MarleenGilsing** [comment]
Sorry for the mess! I think it should be alright now!

**Review by TiBiBa — APPROVED** [review]

**Pull request merged**